client/lib/analytics/index.js

```diff
@@ -345,33 +345,16 @@ const analytics = {
 		recordSignupStartInFloodlight();
 	},
 
-	recordRegistration: function( { flow } ) {
+	recordRegistration: function( { username, userId, flow, type } ) {
+		analytics.identifyUser( username, userId );
```

Comment:
I see a couple of merge conflicts in this branch, but looks very good overall. I think once you resolve conflicts you'll see that we need this call to `identifyUser()` to include the email address too, and it expects an object now.

Reply:
Thanks @jaswrks this is done now.